### PR TITLE
Update to `react-json-tree@^0.16.1`

### PR DESCRIPTION
**Pull request by jtpio**
## References

`@types/react-json-tree` has been deprecated:

image

## Code changes

- [x] Update to `react-json-tree@^0.16.1`
- [x] Remove `@types/react-json-tree`
- [x] Switch to name import: `import { JSONTree } from 'react-json-tree';`

## User-facing changes

Should be none

## Backwards-incompatible changes

None

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -4883,6 +4881,11 @@ csstype@2.6.9:
   resolved "https://registry.yarnpkg.com/csstype/-/csstype-2.6.9.tgz#05141d0cd557a56b8891394c1911c40c8a98d098"
   integrity sha512-xz39Sb4+OaTsULgUERcCk+TJj8ylkL4aSVDQiX/ksxbELSqwkgt4d4RD7fovIdgJGSuNYqwZEiVjYY5l0ask+Q==
 
+csstype@^3.0.10:
```
Comment (fcollonval) [inline yarn.lock:4884]: It looks it can be deduped with the followed entry

File: yarn.lock [hunk 2]
```diff
@@ -868,6 +868,13 @@
     "@babel/helper-validator-option" "^7.14.5"
     "@babel/plugin-transform-typescript" "^7.15.0"
 
+"@babel/runtime@^7.16.7":
```
Comment (fcollonval) [inline yarn.lock:871]: It should be possible to dedupe this one with the following entry

**Review by fcollonval — APPROVED** [review]
Thanks @jtpio 
It looks the yarn.lock file could be simplified. But that can wait.

**Comment by jtpio** [comment]
Right, probably we might want to regenerate the lock file at some point for 4.0 to clean it up.

**Pull request merged**